apps/dev-portal-web/docs/api/cases/invoices/work-with-invoices.ru.mdx

```diff
@@ -0,0 +1,471 @@
+        }
+      ) {
+        id
+        number
+        status
+        organization {
+          id
+          name
+        }
+        property {
+          id
+          address
+        }
+        unitName
+        unitType
+        toPay
+        rows {
+          name
+          toPay
+          count
+          isMin
+          currencyCode
+          sku
+        }
+        createdAt
+        updatedAt
+        createdBy {
+          id
+        }
+        updatedBy {
+          id
+        }
+        deletedAt
+        publishedAt
+        paidAt
+      }
+    }
+    ```
+    ```json
+    {
+      "data": {
+        "obj": {
+          "id": "34b5c321-c086-4540-bca8-18f23580be7d",
+          "number": 4,
+          "status": "draft",
+          "organization": {
+            "id": "e40b5367-49a8-4340-9eed-802538331326",
+            "name": "Тест маркетплейс"
+          },
+          "property": null,
+          "unitName": null,
+          "unitType": null,
+          "toPay": "100.00000000",
+          "rows": [
+            {
+              "name": "Другая услуга",
+              "toPay": "100",
+              "count": 1,
+              "isMin": false,
+              "currencyCode": "RUB",
+              "sku": null
+            }
+          ],
+          "createdAt": "2024-04-11T11:53:51.736Z",
+          "updatedAt": "2024-04-11T12:02:27.642Z",
+          "createdBy": {
+            "id": "810dc183-b9be-4216-aea5-f8237d576c8e"
+          },
+          "updatedBy": {
+            "id": "810dc183-b9be-4216-aea5-f8237d576c8e"
+          },
+          "deletedAt": null,
+          "publishedAt": null,
+          "paidAt": null
+        }
+      }
+    }
+    ```
+</Tabs>
+
+## Публикация счета
+После публикации счет можно увидеть в мобильном приложении и оплатить.
+Если в счете есть позиции с неявной ценой (`isMin=true`), то такой счет невозможно опубликовать.
+Публикация - это [редактирование счета](#редактирование-счета), изменяется поле `status` на значение `published`.
+
+## Ссылка на оплату
+Шаблон ссылки на оплату выглядит вот так:
+```
+https://condo.d.doma.ai/payment-link?su={successUrl}&fu={failureUrl}&i={invoiceId}
+```
+
+| Поле       | Описание                                                                   | Тип/формат          | Пример                                                                    |
+|------------|----------------------------------------------------------------------------|---------------------|---------------------------------------------------------------------------|
+| successUrl | Адрес, на который будет перенаправлен пользователь при успешной оплате     | string, url encoded | https%3A%2F%2Fcondo.d.doma.ai%2Fmarketplace%2Finvoice%2Fpayment%2Fsuccess |
+| failureUrl | Адрес, на который будет перенаправлен пользователь, если оплата не удалась | string, url encoded | https%3A%2F%2Fcondo.d.doma.ai%2Fmarketplace%2Finvoice%2Fpayment%2Ffailure |
```

Comment:
R u sure you want to keep % encoding characters in example?

Reply:
Yes. This is an example. I've shorted strings a bit.